libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailDialog/useScheduledEmail.ts

```diff
@@ -111,14 +117,19 @@ interface UseScheduledEmailResult {
     ) => void;
 
     /**
-     * Status of the scheduled email creation -
+     * Status of the scheduled email creation
      */
     scheduledEmailCreationStatus?: CommandProcessingStatus;
 
     /**
-     * Attachment to be selected by default.
+     * Function that results in saving the existing scheduled email on the backend.
      */
-    defaultAttachment?: ObjRef;
+    handleSaveScheduledEmail: (scheduledEmailToCreate: IScheduledMailDefinition) => void;
```

Comment:
I would rename the `scheduledEmailToCreate` to clearly represent already created schedule. Maybe `scheduledEmail` is enough.

Reply:
yes, my mistake, fixed